packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -298,8 +300,33 @@ const withChunking = (config) => {
     }
 }
 
+// TODO: Once we create a new project for extensibility we'll have the opportunity to better move this utility
```

Comment:
This was another reason why we need our own package, so we have the right place for utils like that that can be used in the sdk and runtime and dev without introducing weird circular dependancies.